docs/developer-guide/api/concrete.ml.torch.hybrid_model.md

```diff
@@ -0,0 +1,544 @@
+
+### <kbd>method</kbd> `__init__`
+
+```python
+__init__(key_path: Path, model_dir: Path, logger: Optional[LoggerStub])
+```
+
+______________________________________________________________________
+
+<a href="../../../src/concrete/ml/torch/hybrid_model.py#L767"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+
+### <kbd>method</kbd> `add_key`
+
+```python
+add_key(key: bytes, model_name: str, module_name: str, input_shape: str)
+```
+
+Add public key.
+
+**Arguments:**
+
+- <b>`key`</b> (bytes):  public key
+- <b>`model_name`</b> (str):  model name
+- <b>`module_name`</b> (str):  name of the module in the model
+- <b>`input_shape`</b> (str):  input shape of said module
+
+**Returns:**
+Dict\[str, str\]
+\- uid: uid a personal uid
+
+______________________________________________________________________
+
+<a href="../../../src/concrete/ml/torch/hybrid_model.py#L689"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+
+### <kbd>method</kbd> `check_inputs`
+
+```python
+check_inputs(
+    model_name: str,
+    module_name: Optional[str],
+    input_shape: Optional[str]
+)
+```
+
+Check that the given configuration exist in the compiled models folder.
+
+**Args:**
+
+- <b>`model_name`</b> (str):  name of the model
+- <b>`module_name`</b> (Optional\[str\]):  name of the module in the model
+- <b>`input_shape`</b> (Optional\[str\]):  input shape of the module
+
+**Raises:**
+
+- <b>`ValueError`</b>:  if the given configuration does not exist.
+
+______________________________________________________________________
+
+<a href="../../../src/concrete/ml/torch/hybrid_model.py#L792"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+
+### <kbd>method</kbd> `compute`
+
+```python
+compute(
+    model_input: bytes,
+    uid: str,
+    model_name: str,
+    module_name: str,
+    input_shape: str
+)
+```
+
+Compute the circuit over encrypted input.
+
+**Arguments:**
+
+- <b>`model_input`</b> (bytes):  input of the circuit
+- <b>`uid`</b> (str):  uid of the public key to use
+- <b>`model_name`</b> (str):  model name
+- <b>`module_name`</b> (str):  name of the module in the model
+- <b>`input_shape`</b> (str):  input shape of said module
+
+**Returns:**
+
+- <b>`bytes`</b>:  the result of the circuit
+
+______________________________________________________________________
+
+<a href="../../../src/concrete/ml/torch/hybrid_model.py#L663"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+
+### <kbd>method</kbd> `dump_key`
+
+```python
+dump_key(key_bytes: bytes, uid: Union[UUID, str]) → None
+```
+
+Dump a public key on the file system.
```

Comment:
where does it go?

Reply:
the `key_bytes` is a stream, I will add this info